cli/azd/extensions/microsoft.azd.extensions/internal/cmd/release.go

```diff
@@ -254,8 +274,28 @@ func runReleaseAction(ctx context.Context, flags *releaseFlags) error {
 			ux.TaskOptions{
 				Title: "Creating Github release",
 				Action: func(spf ux.SetProgressFunc) (ux.TaskState, error) {
+					// Determine patterns to use
+					var patterns []string
+					if flags.artifacts == "" {
+						// Default patterns for both .zip and .tar.gz files
+						localRegistryArtifactsPath, err := internal.LocalRegistryArtifactsPath()
+						if err != nil {
+							return ux.Error, common.NewDetailedError("Failed to get registry path",
+								fmt.Errorf("failed to get registry artifacts path: %w", err),
+							)
+						}
+						basePattern := filepath.Join(localRegistryArtifactsPath, extensionMetadata.Id, flags.version)
+						patterns = []string{
+							filepath.Join(basePattern, "*.zip"),
+							filepath.Join(basePattern, "*.tar.gz"),
+						}
+					} else {
+						// Use explicitly provided pattern or concrete file path
+						patterns = []string{flags.artifacts}
+					}
```

Comment:
This snippet is duplicated a few times here and in `release.go`, is there some light refactoring that could be done here?